website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -581,6 +593,8 @@ curl \
 
 ### Sample Response
 
+**Note:** The `two-factor-conformant` and `assessments-enforced` properties are only returned from Terraform Cloud organizations.
```

Comment:
Updating some of these to be active voice, per our style guide. I also wanted to add this second part to explain why -- the feature just isn't available in TFE yet.

```suggestion
**Note:** Only Terraform Cloud organizations return the `two-factor-conformant` and `assessments-enforced` properties. Health assessments are not available in Terraform Enterprise.
```